src/js/controller/controller.js

```diff
@@ -244,6 +244,22 @@ Object.assign(Controller.prototype, {
 
             _model.change('playlistItem', function(model, playlistItem) {
                 if (playlistItem) {
+                    const { title, image } = playlistItem;
+                    if ('mediaSession' in navigator && window.MediaMetadata && (title || image)) {
+                        try {
+                            navigator.mediaSession.metadata = new window.MediaMetadata({
+                                title,
+                                artist: window.location.hostname,
+                                artwork: [
+                                    { src: image || '' }
+                                ]
+                            });
+                        } catch (error) {
+                            _this.triggerError({
```

Comment:
Catching here does not need to trigger an error. The only failure is setting up a media session. The PLAYLIST_ITEM event will be triggered.